### ISSUE TITLE
Term 1 Lesson 8 CNN Lesson Resource Mistake

### ISSUE THREAD

**Issue by Alexander-Frank** [udacity/sdc-issue-reports#865]
In the file "main.py" line 55 it should be: "fc1 = tf.nn.dropout(fc1, keep_prob)" instead of "fc1 = tf.nn.dropout(fc1, dropout)"

The code will still run fine. However, when calculating batch loss and accuracy, droupout should be set to 1.0 and that will only work when it refers to the variable keep_prob, rather than droupout to my understanding.

Hope that helps.

**Comment by baumanab** [comment]
Thank you for reporting this.  Would you mind providing a link?

**Comment by baumanab** [comment]
Closed as not actionable without updated information.
